README.md

```diff
@@ -19,7 +19,15 @@ $ "$GOPATH/bin/wuzz" --help
 ### Commands
 
 Keybinding                              | Description
-----------------------------------------|------------------------------------------------------------
+----------------------------------------|------------------------------------------------------------	
+<kbd>F2</kbd>                           | URL
```

Comment:
Displaying only the window name isn't really informative for the new users. What about adding `"Jump to "` prefix to the descriptions?